docs/release/release-tasks.md

```diff
@@ -0,0 +1,384 @@
+
+Information can be distributed via: (TBD)
+* `sig-cluster-lifecycle` mailing list
+* Slack
+* Office hours
+* ClusterAPI book
+* ...
+
+Relevant information includes: (TBD)
+* Beta, RC, GA release
+* Start of code freeze
+* Implementation progress
+* ...
+
+Stakeholders are: (TBD)
+* End users of ClusterAPI
+* Contributors to core ClusterAPI
+* Provider implementers
+* ...
+
+## CI Signal/Bug Triage/Automation Manager
+
+### Responsibilities
+
+* Signal:
+    * Responsibility for the quality of the release
+    * Continuously monitor CI signal, so a release can be cut at any time
+    * Add CI signal for new release branches
+* Bug Triage:
+    * Make sure blocking issues and bugs are triaged and dealt with in a timely fashion
+* Automation
+    * Help improve release automation and tools
+    * Maintain and improve release automation, tooling & related developer docs
+
+### Tasks
+
+#### Setup jobs and dashboards for a new release branch
+
+The goal of this task is to have test coverage for the new release branch and results in testgrid.
+
+1. Create new jobs based on the jobs running against our `main` branch:
+    1. Copy `config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main.yaml` to `config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-4.yaml`.
+    2. Copy `test-infra/config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main-upgrades.yaml` to `test-infra/config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-4-upgrades.yaml`.
+    3. Copy `test-infra/config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml` to `test-infra/config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-release-1-4.yaml`.
+    4. Modify the following:
+        1. Rename the jobs, e.g.: `periodic-cluster-api-test-main` => `periodic-cluster-api-test-release-1-4`.
+        2. Change `extra_refs[].base_ref` to `release-1.4`.
+        3. Change `annotations.testgrid-dashboards` to `sig-cluster-lifecycle-cluster-api-1.4`.
+        4. Change `annotations.testgrid-tab-name`, e.g. `capi-test-main` => `capi-test-release-1-4`.
+        5. For periodics additionally: Change interval (let's use the same as for `1.3`).
+        6. For presubmits additionally: Adjust branches: `^main$` => `^release-1.4$`.
+2. Create a new dashboard for the new branch in: `test-infra/config/testgrids/kubernetes/sig-cluster-lifecycle/config.yaml` (`dashboard_groups` and `dashboards`).
+
+Prior art: [Add jobs for CAPI release 1.2](https://github.com/kubernetes/test-infra/pull/26621)
+
+#### [Continuously] Monitor CI signal
+
+The goal of this task is to keep our tests running in CI stable.
+
+1. Add yourself to the [Cluster API alert mailing list](https://github.com/kubernetes/k8s.io/blob/151899b2de933e58a4dfd1bfc2c133ce5a8bbe22/groups/sig-cluster-lifecycle/groups.yaml#L20-L35)
+2. Investigate CI failures reported by mail alerts:
```

Comment:
I don't think the CI manager should be essentially on-call for CAPI tests

If anything, maybe should make this more of a triage role, ie. monitor the test dashboards and file issues for test failures + ensure any regressions identified are marked as release-blocking as applicable

Reply:
I guess my only nit here is 
```suggestion
2. Triage CI failures reported by mail alerts:
```

Reply:
and then I would reword 2. to not say "cause" so it doesn't imply finding the root cause but more identifying if it's a known issue / new issue / regression and what the error causing the failure is (so what are the symptoms more that what's the cause)

Reply:
+1 make this optional with the note that this can help in speeding up the next steps; also there is another "investigate" task below, should we dedup?
+1 to clarify that this person is not on call, but it is his responsibility to surface CI issues to developers so they do not go un-noticed till the extent to delay a release

Reply:
> on-call for CAPI tests

This wasn't meant to be oncall. It's just our experience that it's a lot easier to get pushed by emails and then investigate when there's time (not instantly) vs. having to manually go over dozens of dashboards to figure out which tests are failing.

I'll reframe based on your suggestions to make clear it's not oncall. I would still leave the hint with the alert mailing list as it's just a lot less work this way.

Reply:
@CecileRobertMichon @fabriziopandini  Rephrased, please take a look if the new framing is okay